tests/testthat/test-SurvivalQuantities.R

```diff
@@ -162,4 +162,25 @@ test_that("smoke test for summary(SurvivalQuantities) and autoplot(SurvivalQuant
     expect_true(inherits(p$layers[[2]]$geom, "GeomKm"))
     expect_true(inherits(p$layers[[3]]$geom, "GeomKmTicks"))
 
+
+    #########################
```

Comment:
Maybe nicer to put this into a separate `test_that()` call to avoid this comment dividers?

Reply:
Yeah in that case can be helpful to either prepare the object outside the test and then reuse in below tests. Or if it is fast to execute put I to a test helper function and the execute in each test